src/components/TranslationPanel.js

```diff
@@ -107,19 +113,20 @@ class TranslationPanel extends React.Component {
             AutographaStore.incompleteVerse = incompleteVerseChapter;
             AutographaStore.multipleSpaces = multipleSpacesChapter;      
         })  
-    }
-  
+	}
+
   	render (){
     	let verseGroup = [];
     	const toggle = AutographaStore.toggle;
 
 		for (let i = 0; i < AutographaStore.chunkGroup.length; i++) {
 		let vid="v"+(i+1);
-		verseGroup.push(<div key={i} id={`versediv${i+1}`} onClick={this.highlightRef.bind(this, vid, i)}>
+		verseGroup.push(<div key={i} id={`versediv${i+1}`}  onClick={this.highlightRef.bind(this, vid, i)} style= {{cursor: "text"}}>
```

Comment:
Another stray extra space.

Reply:
Yes I will make sure this unwanted space will not be pushed.